sound/soc/sof/ipc4-topology.h

```diff
@@ -435,6 +435,37 @@ struct sof_ipc4_src {
 	struct sof_ipc4_msg msg;
 };
 
+/*
+ * struct sof_ipc4_asrc_data - IPC data for ASRC
+ * @base_config: IPC base config data
+ * @sink_rate: Output rate for sink module
+ * @asrc_mode: Mask of allowed ASRC features:
+ *             - BITS 0-1 (ASRC mode):
+ *	           a. 0 1 - Playback mode / PUSH mode
+ *	           b. 1 0 - Capture mode / PULL mode
+ *
+ *             - BIT 4 (Disable Jitter Buffer):
+ *                a. 0 - Jitter buffer enabled
+ *                b. 1 - Jitter buffer disabled
```

Comment:
Do we need to describe this in kernel? We just pass this without looking at it.

Reply:
It doesn't impact kernel. We can use ASRC as normal fixed SRC with setting 0x1. It disables the DAI drift measuring and handling. Also handling of this is missing from ASRC IPC4. I'm assuming this is the usage, there's no explicit jitter buffer